storage/tianmu/core/tianmu_attr_exqp.cpp

```diff
@@ -875,19 +875,29 @@ void TianmuAttr::EvaluatePack_BetweenInt(MIUpdatingIterator &mit, int dim, Descr
     mit.NextPackrow();
     return;
   }
-  int64_t pv1 = d.val1.vc->GetValueInt64(mit);
-  int64_t pv2 = d.val2.vc->GetValueInt64(mit);
+  int64_t pv1 = d.val1.vc->GetValueInt64(mit);  // e.g.: between pv1 and pv2
+  int64_t pv2 = d.val2.vc->GetValueInt64(mit);  // e.g.: between pv1 and pv2
+  uint64_t upv1, upv2;                          // used uint64_t to store level 2 encode, ranges from [0, uint64_max]
+  // Only used when rough check = RS_ALL(no numeric data matchs), but table has null, rough check will be reset RS_SOME,
```

Comment:
Your comment description is not very clear and clean. Pls, your precise decriptions thanks.

Reply:
Done.